src/effects.js

```diff
@@ -27,19 +27,27 @@ var
 	rfxtypes = /^(?:toggle|show|hide)$/,
 	rrun = /queueHooks$/;
 
-function raf() {
+function raf( timestamp ) {
 	if ( timerId ) {
 		window.requestAnimationFrame( raf );
-		jQuery.fx.tick();
+		jQuery.fx.tick( timestamp );
 	}
 }
 
+// We need to be using jQuery.now() or performance.now() consistently as they return different
+// values: performance.now() counter starts on page load.
+function getTimestamp() {
```

Comment:
Can we just use `jQuery.now()` for all the browsers that don't have the rAF timestamp? Which browsers are affected?